qiskit_nature/mappers/second_quantization/jordan_wigner_mapper.py

```diff
@@ -44,4 +44,4 @@ def map(self, second_q_op: FermionicOp) -> PauliSumOp:
             pauli_table.append((Pauli((a_z, a_x)), Pauli((b_z, b_x))))
             # TODO add Pauli 3-tuple to lookup table
 
-        return QubitMapper.mode_based_mapping(second_q_op, pauli_table)
+        return QubitMapper.mode_based_mapping(self, second_q_op, pauli_table)
```

Comment:
```suggestion
        return self.mode_based_mapping(second_q_op, pauli_table)
```
This is how this method should be called as an instance method.
The same update needs to be done in all other subclasses of `QubitMapper`